docs/_posts/luca-martial/2022-04-08-bert_embeddings_mengzi_bert_base_zh_3_0.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: model
+title: Chinese Bert Embeddings (from Langboat)
+author: John Snow Labs
+name: bert_embeddings_mengzi_bert_base
+date: 2022-04-08
+tags: [zh, open_source]
+task: Embeddings
+language: zh
+edition: Spark NLP 3.4.2
+spark_version: 3.0
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+Pretrained Bert Embeddings model, uploaded to Hugging Face, adapted and imported into Spark NLP. mengzi-bert-base is a Chinese model orginally trained by Langboat.
```

Comment:
`mengzi-bert-base`  
`Langboat`
(quoted)

Reply:
@luca-martial looks amazing, just quote the model and provider name and we are good!

Reply:
@josejuanmartinez thanks for the review, by quote you mean backtick quote `mengzi-bert-base` or "mengzi-bert-base"?

Reply:
Backticking, so that they look as in the example 

![image](https://user-images.githubusercontent.com/36634572/162701270-60c1ad0e-402e-4f9e-bbd6-cad0bbae40fc.png)